CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+server/chat-plugins/scavenger*.js @xfix
+server/chat-plugins/trivia.js     @xfix
```

Comment:
Is this space aligned? I'd rather not maintain table alignment unless absolutely necessary.

Reply:
Fixed.